docs/Toy/toy/rewrites/lower_toy_affine.py

```diff
@@ -441,17 +441,6 @@ class LowerToAffinePass(ModulePass):
     name = "toy-to-builtin"
 
     def apply(self, ctx: MLContext, op: ModuleOp) -> None:
-        # We define the specific operations, or dialects, that are legal targets for this
```

Comment:
I initially copied this logic from the Toy tutorial, but it broke when I passed in an MLIR file directly, which ought to work. I don't think we really need to replicate this part of the tutorial for the time being